index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Navigation Bar-->
+    <header class="header-container">
+        <div class="left">
+            <a href="/blank"><img id="icons" src="./graphics/open-menu.png" alt="menu"></a>
+            <a href="https://www.youtube.com/"><img class="logo" src="graphics/yt_logo_rgb_light.png" alt="Main-logo"></a>
+
+        </div>
+        <div class="center">
+            <input type="text">
+            <button><img id="icons" src="./graphics/icons8-search-50.png" alt="Search"></button>
```

Comment:
[optional for this project] Ids should be unique. Please use classes instead of id on elements with the same styling in the next projects